monitoring/kafka/alerts.yaml

```diff
@@ -111,6 +111,7 @@ groups:
           > ${maxConsumerLagMessagesWarningThreshold}
       or
       kafka_consumergroup_group_max_lag_seconds{namespace="${namespace}",cluster_name="${cluster}",group!=""}
+          < (1/0)
```

Comment:
not sure what this is supposed to do ?

Reply:
it is needed to filter-out `NaN` / `+Inf` values